mock/imgurSearch.js

```diff
@@ -0,0 +1,1795 @@
+module.exports = {
```

Comment:
You can probably reduce the amount of values in this mock array to 5 max. No need to be so big.